source/_docs/installation/freenas.markdown

```diff
@@ -27,6 +27,8 @@ Install the necessary Python packages:
 # pkg upgrade
 # pkg install -y python37 py37-sqlite3 ca_root_nss
 # python3.7 -m ensurepip
+# pip3 install --upgrade pip
+# pip3 install --upgrade virtualenv
```

Comment:
`virtualenv` should not be needed anymore, since Python3 has `venv` shipped by default.

```
python3 -m venv .
source ./bin/activate
```

should do the trick... right?